dev-requirements.txt

```diff
@@ -8,3 +8,4 @@ bandit>=1.5.1
 pydocstyle>=3.0.0
 pytest>=4.3.1
 pytest-cov>=2.6.1
+chardet>=4.0.0
```

Comment:
`chardet` is a primary dependency and its version should be fixed (because of `dependabot`)